src/groups/mqb/mqbblp/mqbblp_relayqueueengine.cpp

```diff
@@ -286,7 +286,7 @@ void RelayQueueEngine::onHandleConfiguredDispatched(
             << downStreamParameters << ", but assuming success.";
     }
 
-    BALL_LOG_INFO_BLOCK
+    BALL_LOG_TRACE_BLOCK
     {
         mqbcmd::RoundRobinRouter outrr(d_allocator_p);
         context->d_routing_sp->loadInternals(&outrr);
```

Comment:
This `loadInternals` might be called for each consumer rapidly and might degrade overall performance when we have thousands of consumers and still continue to add new.

Reply:
Could we make this a debug level instead?

Reply:
How about `BALL_LOGTHROTTLE_INFO_BLOCK`?

Reply:
@hallfox changed trace->debug

Reply:
@hallfox will use throttle info block instead, so we have something in logs without changing default severity